File: prime.py
"""
* author Cak Juice
"""

import time

current_milli_second = lambda: int(round(time.time() * 1000))

start_time = current_milli_second()

primes = []
limit = 25000

for n in range(2, limit + 1):
	is_prime = True
	for div in range(2, n):
		if n % div == 0:
			is_prime = False
			break

	if is_prime:
			primes.append(n)

end_time = current_milli_second()

print(primes)

print('Start Time:', start_time)
print('End Time:', end_time)
print('Selisih:', end_time - start_time, 'ms')

File: prime.php
<?php

/*
 * author Cak Juice
 */

function currentMilliSecond() {
	return round(microtime(true) * 1000);
}

$startTime = currentMilliSecond();

$primes = [];
$limit = 25000;

for ($n=2;$n<=$limit;$n++) {
	$isPrime = true;
	for ($div=2;$div<$n;$div++) {
		if ($n % $div == 0) {
			$isPrime = false;
			break;
		}
	}

	if ($isPrime) {
		array_push($primes, $n);
	}
}

$endTime = currentMilliSecond();

print_r($primes);

echo 'Start Time: ' . strval($startTime) . "\n";
echo 'End Time: ' . strval($endTime) . "\n";
echo 'Selisih: ' .strval($endTime - $startTime) . "ms\n";